app/utils.py

```diff
@@ -76,7 +75,7 @@ def get_local_month_from_utc_column(column):
     """
     return func.date_trunc(
         "month",
-        func.timezone(getenv("TIMEZONE", "America/New_York"), func.timezone("UTC", column))
+        func.timezone(getenv("TIMEZONE", "UTC"), func.timezone("UTC", column))
```

Comment:
We may be able to skip the `getenv` call now, as it should be UTC for anybody who runs this application besides us, as well.